src/Service/Startup.cs

```diff
@@ -287,13 +287,15 @@ private void AddGraphQLService(IServiceCollection services)
         public void Configure(IApplicationBuilder app, IWebHostEnvironment env, RuntimeConfigProvider runtimeConfigProvider)
         {
             bool isRuntimeReady = false;
+            FileSystemRuntimeConfigLoader fileSystemRuntimeConfigLoader = (FileSystemRuntimeConfigLoader)runtimeConfigProvider.ConfigLoader;
```

Comment:
Can we create a task that notes we'll need to clean this up in the future since there will be instances where we won't have the concept of a **filename** to use in the log message on line 298/exception on 309, so the message will need to be refactored/reworked to provide contextually accurate information.

Reply:
Sure I will create a task to clean this up. https://github.com/Azure/data-api-builder/issues/1591#issue-1810334013